app/src/services/GitHubMarkdownService.php

```diff
@@ -2,14 +2,16 @@
 
 use GuzzleHttp\Client;
 use GuzzleHttp\Exception\RequestException;
+use SilverStripe\Core\Convert;
+
 
 /**
  * A GitHub service for communicating with the GitHub API to render Markdown. Uses Guzzle as the
  * transport method.
  *
  * @package mysite
  */
-class GitHubMarkdownService extends Object
+class GitHubMarkdownService
```

Comment:
Not sure how this is instantiated, but we should probably add the Injectable trait so it can be used as a singleton